sdk/storage/azure-storage-blobs/CMakeLists.txt

```diff
@@ -32,9 +32,9 @@ if(FETCH_SOURCE_DEPS)
         add_subdirectory(${oneFolder} EXCLUDE_FROM_ALL)
     endforeach()
 elseif(NOT AZ_ALL_LIBRARIES)
-  find_package(azure-storage-common-cpp "12.3.0-beta.1" CONFIG QUIET)
+  find_package(azure-storage-common-cpp CONFIG QUIET)
```

Comment:
If we revert the latest change, shouldn't this be `12.2.4`?

Reply:
It can be, but will it compile, does it have all the API that Blobs are using? If not, there's no much difference to having no version specified?